app/lib/forms/choose_your_npq.rb

```diff
@@ -2,18 +2,41 @@ module Forms
   class ChooseYourNpq < Base
     include Helpers::Institution
 
-    attr_accessor :course_id
+    QUESTION_NAME = :choose_your_npq
```

Comment:
At it's core, I'm not sure why we're making this change and adding in the LEGACY_NAME_MAPPING system. What does that accomplish?

We're not sending this across to ECF, we're not using it anywhere. Why are we not able to continue using the result from `courses` to populate the list of options and use the ID->Name pairing from the record for the radio value->name combo?

Reply:
Maybe im missing something but i had great difficulty with the auto-magic of the question generation and layout. IE this is more to do with headings layout of the question.

Reply:
Going to PM you about this

Reply:
I think i misunderstood what you meant when i was fixing it;

So issue we have is we need to identify the course and the question button only allows us to set one value on the radio button? previously we had the text value sent across aswell which was using the course name. So we cant change the course name in the DB as i understand but have to change the value which is currently linked to the course through the UI and ID? 

Going forward with this question format and how we are storing values/ answers etc i feel we need to make a decision to how this is achieved or maybe it already has. @Lockyy your raw application data makes a lot of sense but then we need to agree what is stored there? Does it matter that the ui is displaying different information to the actual course name? 

Im not sure using course ID limited to the DB is the best option as felt a code smell linking the code to a set order in the db? I feel the course name is unlikely to change that is offered by the provider and even if it does a little this should then be added as a new course so we keep a record of the previous choices rather than changing the existing ones.

Reply:
I think we've discussed this separately but wanted to note, the raw_application_data is just the store as it was at the point of submission to act as an audit trail for what the user entered. It is not intended to be anything more complicated than an exact record of the store.